src/model.ts

```diff
@@ -424,14 +489,13 @@ class FileBrowserModel implements IDisposable {
     }, error => console.error(error));
   }
 
-
   private _max_upload_size_mb = 15;
-  private _selectedIndices: number[] = [];
   private _contentsManager: IContentsManager = null;
   private _sessionIds: ISessionId[] = [];
   private _sessionManager: INotebookSessionManager = null;
   private _model: IContentsModel = null;
   private _kernelSpecs: IKernelSpecId[] = [];
+  private _selection: { [key: string]: boolean; } = null;
```

Comment:
If I initialize something in the constructor, I explicitly *do not* initialize it inline as well. That way when I'm reading the code, I can look at see "oh, this must be initialized in the ctor"